### PR TITLE
CNF-11184: remove seed image even if it is not mounted

### PR DESCRIPTION
**Pull request by sudomakeinstall2**
Remove seed image even if it is not mounted as part of `UnmountAndRemoveImage`

### CODE REVIEW

File: lca-cli/ops/ops.go [hunk 1]
```diff
@@ -281,6 +269,22 @@ func (o *ops) UnmountAndRemoveImage(img string) error {
...
+	}
+	if !mounted {
+		return o.removeImage(img)
+	}
+	if _, err := o.hostCommandsExecutor.Execute(
+		"podman", "image", "umount", img,
+	); err != nil {
+		return fmt.Errorf("failed to unmount image: %w", err)
+	}
+	return o.removeImage(img)
```
Comment (omertuc) [inline lca-cli/ops/ops.go:285]: Would this be equivalent?

```suggestion
	mounted, err := o.IsImageMounted(img)
	if err != nil {
		return fmt.Errorf("failed to check if image is mounted: %w", err)
	}

	if mounted {
		if _, err := o.hostCommandsExecutor.Execute(
			"podman", "image", "umount", img,
		); err != nil {
			return fmt.Errorf("failed to unmount image: %w", err)
		}
	}

	return o.removeImage(img)
```

**Review by omertuc — CHANGES_REQUESTED** [review]

**Review by jc-rh — APPROVED** [review]

**Pull request merged**